src/modules/secrets/dtos/create-Secrets.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty } from "@nestjs/swagger";
```

Comment:
Please, do not name the files like this. Either go for camelCase like "createSecrets" or kebab-case (better) as in this file but change it to all lowercase: **create-secrets.dto.ts**